guides/v2.1/cloud/cdn/configure-fastly.md

```diff
@@ -149,17 +159,22 @@ Make sure to use the correct credentials.
 Complete the following configuration steps in Staging and Production environments:
 
 1.	Log in to your local Magento Admin as an administrator.
-2.	Click **Stores** > **Settings** > **Configuration** > **Advanced** > **System**.
-3.	In the right pane, expand **Full Page Cache**.
+
+1.	Click **Stores** > **Settings** > **Configuration** > **Advanced** > **System**.
+
+1.	In the right pane, expand **Full Page Cache**.
 
 	![Expand to select Fastly]({{ site.baseurl }}/common/images/cloud_fastly_menu.png){:width="650px"}
 
-4.	For **Caching Application**, uncheck the **Use system value** checkbox and select **Fastly CDN** from the drop-down list.
+1.	For **Caching Application**, uncheck the **Use system value** checkbox and select **Fastly CDN** from the drop-down list.
 
 	![Choose Fastly]({{ site.baseurl }}/common/images/cloud-fastly_enable-admin.png){:width="550px"}
-5.	Expand **Fastly Configuration**. You can then [choose caching options](https://github.com/fastly/fastly-magento2/blob/master/Documentation/CONFIGURATION.md#configure-the-module).
-6.	When you're done, click **Save Config** at the top of the page.
-7.	Clear the cache according to the notification. After you have cleared the
+	
+1.	Expand **Fastly Configuration**. You can then [choose caching options](https://github.com/fastly/fastly-magento2/blob/master/Documentation/CONFIGURATION.md#configure-the-module).
```

Comment:
There is no need for "You can..." in steps. They should be quick, do this kind of statements. If you need to expand, you can go to the next line? But this seems like it should say: 
"Expand **Fastly Configuration** and choose caching options."

Reply:
So just FYI -- this PR was just about the credentials sections. I have rewritten the remaining sections of the configure-fastly.md file. Will be submitting in another PR.  I thought I had mentioned this earlier. Apologies for the unnecessary editorial review.